spring-batch-docs/asciidoc/spring-batch-integration.adoc

```diff
@@ -315,58 +313,57 @@ represented by the job parameter "input.file.name":
 ----
 
 
-The main points of interest here are injecting the value of
+The main points of interest in the preceding example are injecting the value of
 `#{jobParameters['input.file.name']}`
 as the Resource property value and setting the `ItemReader` bean
-to be of __Step scope__ to take advantage of
-the late binding support which allows access to the
+to have __Step scope__, Setting the bean to have Step scope takes advantage of
```

Comment:
Why is setting capitalized?

Reply:
I meant to make it be two sentences. The comma before "Setting" should be a period. Thans for catching it.